x-pack/platform/plugins/shared/fleet/server/services/agents/crud.ts

```diff
@@ -185,7 +185,9 @@ export async function getAgentTags(
       },
     });
     const buckets = result.aggregations?.tags.buckets;
-    return buckets?.map((bucket) => bucket.key) ?? [];
+    return (buckets?.map((bucket) => bucket.key) ?? []).toSorted((a, b) =>
+      a.toLowerCase().localeCompare(b.toLowerCase())
+    );
```

Comment:
I looked at bit at updating the ES query to see if we could get case insensitive sorting here in the agg:
https://github.com/elastic/kibana/blob/main/x-pack/platform/plugins/shared/fleet/server/services/agents/crud.ts#L175-L185

But came up short of good approaches to it.

Reply:
I poked around at the docs and indeed case-insensitive sort on an aggregation is tricky. this approach LGTM